src/adapters/osm_schedule.js

```diff
@@ -31,10 +31,27 @@ function nextTransitionTime(seconds, nextTransitionDate) {
   return false
 }
 
+function hourToDate(hour) {
+  return new Date(new Date().toDateString() + ' ' + hour)
+}
+
+function translateOpeningHours(hours) {
```

Comment:
(Law of triviality) it's more a localisation than a translation

Reply:
renamed toLocaleOpeningHours